### DISCUSSION TITLE
New Page Ideas

### DISCUSSION THREAD

**Discussion by kevdagoat** [hack-technicolor/hack-technicolor#18] [Ideas]
I was thinking about some new pages to really explain how this modem works.

Here are my ideas:
- Web Interface - @Ansuel could write a small amount on how the web interface is structured and how the backend works
- Installing your own Packages - A guide on how to use [brcm63xx-tch](https://github.com/bolamn/brcm63xx-tch-src)

Edit:
- [ ] Serial Console
- [x] The boot process
- [ ] Web Interface
- [ ] Writing Transformer Mappings

**Comment by kevdagoat** [comment]
Others could be:
- How to create an init.d script to prevent firmware upgrades from breaking root access
- How to write transformer mappings
- An overview of the boot process

**Comment by LuKePicci** [comment]
All good, That one for transformer mappings would be great. 

Da: Kevdagoat
Inviato: giovedì 11 luglio 2019 04:00
A: kevdagoat/hack-technicolor
Cc: Subscribed
Oggetto: Re: [kevdagoat/hack-technicolor] New Page Ideas (#18)

Others could be:
• How to create an init.d script to prevent firmware upgrades from breaking root access
• How to write transformer mappings
• An overview of the boot process

**Comment by nutterthanos** [comment]
@kevdagoat so are we going to show the generic boot process or a device specific boot process?

**Comment by LuKePicci** [comment]
Homeware boot process is neither device specific nor firmware specific. it’s still the same even in old 10.x releases.

Da: nutterthanos
Inviato: venerdì 12 luglio 2019 03:22
A: kevdagoat/hack-technicolor
Cc: Luca Piccirillo; Comment
Oggetto: Re: [kevdagoat/hack-technicolor] New Page Ideas (#18)

@kevdagoat so are we going to show the generic boot process or a device specific boot process?

**Comment by nutterthanos** [comment]
@LuKePicci so its generic with any homeware firmware?

**Comment by LuKePicci** [comment]
From bootloader stage 2 unpacking to kernel init, yes, absolutely. Looks like there are some strage differences in some models where the bootloader stage 1 is also decrypting stage 2, but this is totally irrelevant here.

________________________________
Da: nutterthanos <notifications@github.com>
Inviato: Friday, July 12, 2019 1:03:39 PM
A: kevdagoat/hack-technicolor
Cc: Luca Piccirillo; Mention
Oggetto: Re: [kevdagoat/hack-technicolor] New Page Ideas (#18)


@LuKePicci<https://github.com/LuKePicci> so its generic?

**Comment by kevdagoat** [comment]
@LuKePicci I’d think you would be the perfect person to do that boot process guide :)

**Comment by LuKePicci** [comment]
Maybe after summer holidays

**Comment by Ansuel** [comment]
Ahahah yhea not a priority

Il Sab 13 Lug 2019, 12:38 Luca Piccirillo <notifications@github.com> ha
scritto:

**Comment by flywire** [comment]
Serial Console

**Comment by kevdagoat** [comment]
@flywire

Sounds good!

Used to explain the output from the serial console. 

Two sections: MIPS and ARM

**Comment by flywire** [comment]
Start with install including opening case of each model and the pads.

**Comment by kevdagoat** [comment]
Good point

**Comment by nutterthanos** [comment]
Any progress?

**Comment by LuKePicci** [comment]
Now doing some page splitting, give a look to latest branch, share feedbacks if any.

---------------

Da: nutterthanos
Inviato: mercoledì 25 dicembre 2019 23:41
A: kevdagoat/hack-technicolor
Cc: Luca Piccirillo; Mention
Oggetto: Re: [kevdagoat/hack-technicolor] New Page Ideas (#18)

Any progress?

**Comment by benryanau** [comment]
(new recruit, limited by health issues but having a go :))
RE: J5 UART/Serial/Console - It'd be good to have this part fleshed out, the hardware side of things is pretty barebones and I can contribute here I think. I'll need help and guidance in getting info into the docs however, it's my first time with the github/RTD process.
Email is out for me at the moment pretty much but I can manage limited browser access at times.

I'll try raising an issue for it :)
